src/Lecture1.hs

```diff
@@ -119,4 +137,15 @@ and lower than 6 elements (4, 5, 6, 7, 8 and 9).
 
 🕯 HINT: Use recursion to implement this function.
 -}
-lowerAndGreater n list = error "TODO"
+lowerAndGreater :: Integer -> [Integer] -> [Char]
+lowerAndGreater n list = 
+    let greater = 0
+        lower = 0
+    in comp n list greater lower
```

Comment:
I see that you pass `n` during recursive calls but never change it. You can remove this extra argument in that cases. Functions inside `where` can see all the arguments of the main function (because it's in the scope) so you don't need to pass `n` to the `comp` function.

As a consequence, this will remove all GHC warnings in this function 🙂

Reply:
Oh, I didn't notice.
Thanks!